.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "Carthage/Checkouts/promises"]
-	path = SampleMacOS/Carthage/Checkouts/promises
```

Comment:
I think the path here is wrong, it should be
```
[submodule "Carthage/Checkouts/promises"]
        path = Carthage/Checkouts/promises
        url = https://github.com/google/promises.git
```

Reply:
Thanks, I did update the SampleMacOS to `sdk-ble-swift` but I didn't push that change. I'll definitely try the path as you wrote it. Thanks @oldsquarehead